### ISSUE TITLE
0.2.7 fails when calling .zip()

### ISSUE THREAD

**Issue by symbyte** [mapbox/shp-write#37]
the call fails with the following stack trace: 

```
writer.parts is not a function
    at write (node_modules/shp-write/shpwrite.js:12510:24)
    at eval (node_modules/shp-write/shpwrite.js:12570:13)
    at Array.forEach (native)
    at Module.module.exports (node_modules/shp-write/shpwrite.js:12568:10)
    at GeoConvert.toShapefile (dist/app/main/util/geo-convert.service.js:9:3495)
    at Object.it (spec-compiled/main/geo-convert.service.spec.js:61:44)
    at ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:332:29)
    at ProxyZoneSpec.onInvoke (node_modules/zone.js/dist/proxy.js:106:44)
    at ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:331:35)
    at Zone.run (node_modules/zone.js/dist/zone.js:225:44)
    at Object.<anonymous> (node_modules/zone.js/dist/jasmine-patch.js:123:51)
    at ZoneQueueRunner.jasmine.QueueRunner.ZoneQueueRunner.execute (node_modules/zone.js/dist/jasmine-patch.js:151:43)
    at ZoneQueueRunner.jasmine.QueueRunner.ZoneQueueRunner.execute (node_modules/zone.js/dist/jasmine-patch.js:151:43)
    at ZoneDelegate.invokeTask (node_modules/zone.js/dist/zone.js:365:38)
    at Zone.runTask (node_modules/zone.js/dist/zone.js:265:48)
    at drainMicroTaskQueue (node_modules/zone.js/dist/zone.js:497:36)
    at ZoneTask.invoke (node_modules/zone.js/dist/zone.js:437:26)
```

**Comment by haoliangyu** [comment]
This error is particularly related to `POINT` type because `points.js` doesn't have a `parts()` function.

**Comment by tmcw** [comment]
Fixed in #39
